google-cloud-spanner-hibernate-dialect/src/test/java/com/google/cloud/spanner/hibernate/SpannerTableExporterTests.java

```diff
@@ -139,13 +139,13 @@ public void omitCreatingPreexistingTables() throws IOException, SQLException {
 
     assertThat(statements).containsExactly(
         // This omits creating the Employee table since it is declared to exist in metadata.
-        "START BATCH DDL",
-        "create table hibernate_sequence (next_val INT64) PRIMARY KEY ()",
-        "create index name_index on Employee (name)",
+        "START BATCH DDL;",
```

Comment:
Hibernate now uses the `;` separator as the default when generating DDL scripts: https://in.relation.to/2021/06/15/hibernate-orm-552-release/